addons/account/views/res_users_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="res_users_form_view" model="ir.ui.view">
+        <field name="name">res.users.form.sale.inherit</field>
```

Comment:
ZIzisse not sale, zizisse account ! To check other xml ids, this is crucial for the feature.